HydrateOrDiedrate/src/Piping/HandPump/BEBehaviorHandPumpAnim.cs

```diff
@@ -0,0 +1,68 @@
+using Vintagestory.API.Client;
+using Vintagestory.API.Common;
+using Vintagestory.API.Datastructures;
+using Vintagestory.API.MathTools;
+using Vintagestory.GameContent;
+
+namespace HydrateOrDiedrate.Piping.HandPump
+{
+    public class BEBehaviorHandPumpAnim : BEBehaviorAnimatable
+    {
+        public bool IsPumping;
```

Comment:
should not be publicly `settable`

```suggestion
        public bool IsPumping { get; protected set; }
```